README.md

```diff
@@ -343,6 +346,78 @@ For example, if we named the above XML launch script, `record_all.launch.xml`:
 $ ros2 launch record_all.launch.xml
 ```
 
+You can also invoke the `play` and `record` functionalities as nodes. Nodes `player` and `recorder` are available in `rosbag2_transport` package. The advantage to use this invocation strategy is that the Python layer handling the `ros2 bag` CLI is completely skipped.
+
+```python
+import launch
+
+def generate_launch_description():
+    return launch.LaunchDescription([
+        launch.actions.Node(
+            package='rosbag2_transport',
+            executable='player',
+            name='player',
+            output="screen",
+            parameters=["/path/to/params.yaml"],
+        )
+    ])
+```
```

Comment:
Need to add a short example of the `params.yaml` config file. We can put a few essential lines for the idea of the syntax and how it should look like. Since we support partial parameters specification in yaml config it is will be a valid example to provide just a few parameters in it.

Reply:
If you change the storageoptions.uri parameter, will it start writing under a new filename or are those parameters only used at startup? This could be really useful say for having named recordings.

Reply:
May we link rosbag2_transport/test/resources/player_node_params.yaml and siblings to provide a full reference instead? Or: is a full reference file with all options and defaults needed? The linked one has purposely chosen bad or absurd params in it since it's meant as a test asset.

Reply:
We can provide a link on test's yaml file but the short builtin example in addition will be better IMO.